### ISSUE TITLE
Team - Design Studios - Ideation Exercises and Output

### ISSUE THREAD

**Issue by iamjoshfrank** [revelry-foundation/nightingale#15]
## Background
When the research phase [milestone](https://github.com/revelry-foundation/nightingale/milestone/6) is completed and the Prep Work of #39 has been done, we're ready to move on to actually running the Design Studio exercises and capturing our output. 

### Resources 
- How to run a [design studio](https://www.mural.co/blog/how-to-run-a-design-studio-remotely-with-mural)
- Mural [Board](https://app.mural.co/t/nightingale1567/m/nightingale1567/1590690171105/d56b4e0fc5ecb3d6bfc523ba5e8007282b30e731)
- Validation [Board](https://docs.google.com/spreadsheets/d/1ih9t6cCNgddM-vwEediRvrWp7QaZ1DbxvVQ1FqEWzkM/edit?usp=sharing)
- Guided Work shop [presentation](https://docs.google.com/presentation/d/1_ydEOjPx_ra3FW6_Xz9JpvIjHjjIFx_Jg0atx_2_6ug/edit#slide=id.g873215e515_0_0)

## Scenario: Design Studio Itinerary 
- [x] Overview / Introduction: [Slide Show](https://docs.google.com/presentation/d/1_ydEOjPx_ra3FW6_Xz9JpvIjHjjIFx_Jg0atx_2_6ug/edit#slide=id.g873215e515_0_0)
- [x] Research Overview: [Key Insights Doc](https://docs.google.com/document/d/1Mw05U2NIXxO2--ZoITQJGTwE7jyZBtlblybkIkyF1ko/edit)
- [x] Round 1: Ideation:
	- [x] Brain Storm
	- [ ] Sketch 
	- [x] Highlights
- [x] Round 2: Ideation:
	- [x] Brain Storm
	- [x] Sketch 
	- [x] Highlights
	- [x] Vote

**Comment by Mcscott142** [comment]
EOD Update: We made it through the BrainStorm portion, had some really good conversations and nice output. We'll continue on to pairing up and doing some Sketches tomorrow. [Mural Board](https://app.mural.co/t/nightingale1567/m/nightingale1567/1590690171105/d56b4e0fc5ecb3d6bfc523ba5e8007282b30e731)

**Comment by jeremybrady** [comment]
### Thank you for joining our round 1 of 2 of our first design studio team! 
@CheetoMao @Mcscott142 @HauwaAguillard @vegasgeek @BenRongey @alinelea 

Your work leading up to today and ongoing will help inform all the work we do, informing both tracks in an equal effort to create a bad a@#! MVP. So again, thank you for being troopers.

**1.** Calling out our channel here #nightingaledesignstudio for discussions
**2.** Calling out [our Mural board for ideation ](https://app.mural.co/invitation/mural/nightingale1567/1590690171105?sender=jeremy4340&key=d745f756-9443-4b00-a799-8e21f3ab29b3)
**3.** [Calling out our design validation board here](https://docs.google.com/spreadsheets/d/1ih9t6cCNgddM-vwEediRvrWp7QaZ1DbxvVQ1FqEWzkM/edit?usp=sharing)
**4.** Calling out our [guided workshop presentation](https://docs.google.com/presentation/d/1_ydEOjPx_ra3FW6_Xz9JpvIjHjjIFx_Jg0atx_2_6ug/edit?usp=sharing)

--

### Background; 
**Key Insights and assumptions for reference**

Interviewed; 
*   [Jennifer Galvin](https://docs.google.com/document/d/1hMy7E6bkqjbJnC3w42kOp4biCRIkPXYBxm04nCBTF5c/edit?usp=sharing)
*   [Dr. Danielle Thomas](https://drive.google.com/drive/folders/1LzMuFlfrKEgaBv5_JL-km_C9YPoH9ysJ?usp=sharing)
*   [Allen Square](https://drive.google.com/drive/folders/1gF6DqwBM2jvc6ZjTgRSG3V6tMWA4aYbm?usp=sharing)
*   [Mikal Matton](https://drive.google.com/drive/folders/11dArUzjSgJj0DZdazzit-Q8td81hX6wM?usp=sharing)
*   [Veena Billioux](https://docs.google.com/document/d/1vGH_3vNtkbDcy9YPdzILc3mMx2LoSznAJwtS-dcwB_s/edit?usp=sharing)
*   [Michael A.V.](https://drive.google.com/drive/folders/1gAhMtFfc_MCnyo2hhCEiEkJxNAlJ2R9B?usp=sharing)
*   [Scott Teesdale](https://drive.google.com/drive/folders/1zUMhsq8PFh5W1uODCUxa806s-pM8sO-m?usp=sharing)
*   [Christopher O'Shea](https://drive.google.com/open?id=1dN16SeRS53ldGFXQQnJtXXGwuOLLKU23psUlFIxdlEE) 
*   [Josh Cox](https://www.linkedin.com/in/joshua-cox-0387598/)

...with in our customer pool we’ve observed the following



*   **Public health**
    *   **Management**
        *   Is a state and local issue, rarely a federal one
    *   **Goal**
        *   Progress against Covid-19 is intangible based on the goal of public health and therefore hard to incite public action
        *   Contact tracing goal is to efficiently get at reducing transmission by half though; so there is some expected lost to followup
    *   **Empower**
        *   Needs to be empower, so PH officials so they can do what they’re good at
    *   **Contact Tracing is only successful when; **
        *   Directed by public health officials at the state and local level 
        *   Directed by trusted boots on the ground community leaders
    *   **“Army” of educated and empowered tracers needed**
        *   Education of tracers needed on how to go out into the community and perform tracing.
*   **Awareness**
    *   **Comms format;** 
        *   Simple data visualizations are needed to provide context and address cultural nuance
    *   **First priority; [Preventive measures](https://docs.google.com/document/d/1vGH_3vNtkbDcy9YPdzILc3mMx2LoSznAJwtS-dcwB_s/edit#bookmark=id.jfbimwql5pzs)—reduce at risk behaviors**

        This is about buying time not flattening the curve to make this go away

        *   Recommendations; “Here are ten basic things that everyone should be doing all the time, starting today:”
            *   Emphasize—What does 6’ look like?— people don’t know. 
                *   If you go to a small event, go to the beach for a run—helping people put visuals to these basic numbers is important.
                *   It’s going to be more important as we go into the summer—it’s hot, and people are going to want to go out, especially large multi-generational homes
*   **Second priority;** Its O.K. To tell someone you have it or you should get tested. Diseases have stigmas—people are private around their health status or health history, there are serious privacy concerns**. **Diseases have stigmas—people are private around there health status or health history, there are serious privacy concerns
        *   
    *   **Third priority; Post Positive Test Result Measures—Provide a support network**
        *   
*   **Adoption**
    *   Trace 60%+
    *   Best way to get adoption (contact tracing tool) is to do this in the open source—your prototype that drives equity that is open source and uses open data
    *   Reach out and mobilize youth within at risk communities
    *   Reach out and mobilize community leaders;
        *   Barbers, salons, faith based pastors/ministers
    *   Be Clear, concise and simple information
    *   Making this open source, we’ll get more innovation from the community
    *   Message doesn’t need to change, but how we get it out there needs to be catered for the different segments
*   **At Risk Communities**—**Segmentation**

15 percent of all Covid-19 cases fit within the at risk category
*   **Pre-existing conditions;**
    *   Obesity
    *   Chronic lung disease/asthma
    *   Heart Disease
    *   Liver Disease
    *   Diabetes
    *   Kidney Disease or is Dialysis
    *   Immunocompromised
    *   Cardiac issues
        *   Positive Covid-19 patients w/ Cardiac issues showing long-term side effects 2 months on
    *   Severe Obesity
    *   **African-Americans and people of color **
        *   Don’t trust technology
        *   Don’t trust gov’t
        *   Don’t have access to the latest technology
        *   Disproportionately affected by Covid-19 and make up a significant portion of the at risk population
        *   Have large multi-generational household(s)
            *   Alot of multi-generational homes have one person, can be the source of information
*   **At Risk Communities**—**Segmentation**
    *   **Low income communities**
        *   Don’t trust technology
        *   Don’t trust gov’t
        *   Don’t have access to the latest technology
*   **At Risk Communities**—**Segmentation**
    *   **Essential worker **(Healthcare, Grocery, Pharmacy, etc.)
        *   Get a stronger dose of Covid-19 due to proximity
    *   **Veterans**
        *   Those with PTSD and substance abuse issues. 

            Exploring segment needs more data.

    *   **Mental Health Conditions**
        *   Mental health issues act as a barrier to manage preventive measures
        *   Exploring segment need more data 
*   **Call-in**
    *   Covid patients will have a hard time when they’re asked to call-in. 
        *   Will have a hard time waiting on hold because they’re exhausted
        *   Will have a hard time talking because they have a respiratory problem
*   **Data Sources of Truth**
    *   Partner w/ city to get 311 logs, understand the scale from crisis to recovery
        *   Example; You can follow the arc of a crisis by tracking logs; From tree down, to floods, to when my power comes back on, when my cable isn't on, to eventually when the crisis calms down.
*   **Technology and Economics (Need more data here)**
    *   3.3% of Louisiana households w/ no phone access
    *   Most marginalized communities don’t have access to WiFi, unlimited cell phone data plans, and we expect to see increases in lack of access as time goes on
        *   Jobless rates at 30%

**Problem Statement**

We have observed that the need for and the use of contact tracing efforts aren’t being effectively communicated out and delivered against for the most at risk communities within Louisiana. 

This is limiting our state's goal to contain the spread and transmission of COVID-19 by half. 

So how might we, working with public health officials at the state and local level and community leaders with “boots on the ground”; 

Increase awareness for the need of tracing within at risk communities



*   Improve tracing through a creation of a tool that both educates and empowers trusted tracers 
*   Create a tool that allows tracers to privately and securely to go into their communities and capture the status of individuals who are at risk 
*   So that we reduce the transmission of COVID-19 by half.

**Broad Customer Hypothesis**

We believe that creating an intake form that identifies an individual's risk factors and gives them resources for Community leaders who reach out to their communities and the younger generation within multi-generational family homes who have an at-risk family member will result in increased awareness and adoption of tracing efforts. We’ll know we’re right when we see a 60%+ level of tool *adoption; individual use of the intake tool and a decrease of COVID-19 transmission by half.

We believe that creating a helpful how to guide on preventive measures residents can take to avoid the contracting and transmission of COVID-19 for Community leaders who reach out to their communities and the younger generation within multi-generational family homes who have an at-risk family member will result in increased awareness and adoption of tracing efforts. We’ll know we’re right when we see a 60%+ level of tool *adoption; individual use of the intake tool and a decrease of COVID-19 transmission by half.

**Comment by jeremybrady** [comment]
Good morning team. 
@CheetoMao @Mcscott142 @HauwaAguillard @vegasgeek @BenRongey @alinelea

### We're now at round 2 of our design studio. 
[I look forward to building on all of our great ideas thus far in MURAL (see the link here)](https://app.mural.co/invitation/mural/nightingale1567/1590690171105?sender=jeremy4340&key=d745f756-9443-4b00-a799-8e21f3ab29b3)
As always thank you for participating and bringing y'alls A-game

**Ahead of meeting:**
[Download and or draw a 6-Up storyboard sheet, and have a pen handy](https://drive.google.com/file/d/1rqtc-y6XwwB4ZXzjseyHX0FPcknNKHDf/view)

[Workshop flow](https://www.google.com/url?q=https://docs.google.com/presentation/d/1_ydEOjPx_ra3FW6_Xz9JpvIjHjjIFx_Jg0atx_2_6ug/edit%23slide%3Did.g808d902bca_0_42&sa=D&ust=1591193872852000&usg=AOvVaw1VerztBaBE2TXt_n6mdqfV)
**Agenda:**
- Review Synthesized Feedback 
- Sketching — refining our ideas into storyboards ([see 6-Up](https://drive.google.com/file/d/1rqtc-y6XwwB4ZXzjseyHX0FPcknNKHDf/view))
- Discuss, Critique & Vote
- Define target MVP to test

**Comment by jeremybrady** [comment]
<table>
  <tr>
   <td><strong>We believe that (doing/building/creating this)</strong>
   </td>
   <td><strong>For this user/persona</strong>
   </td>
   <td><strong>Will result in (this outcome)</strong>
   </td>
   <td><strong>We’ll know we’re right when we see (this signal / metric)</strong>
   </td>
  </tr>
  <tr>
   <td>We will drive awareness of our contact tracing tool “Know Your Status, Share Your Status” by creating a multilingual Downloadable Open Source “We Will &lt;fill in the blank cultural activity> Again.” social video and print campaign, that can be turned into a meme and shared out to social… 
   </td>
   <td>Community leaders who reach out to their communities.
<p>
POC—younger generation within multi-generational family homes who have an at-risk family member
   </td>
   <td>Increased in awareness and adoption of tracing efforts
   </td>
   <td>A 60%+ level of tool *adoption; individual use of the intake tool and a decrease of COVID-19 transmission by half.
   </td>
  </tr>
  <tr>
   <td>We will create a “Know Your Status, Share Your Status” Web App, that will include a… 
<p>
<strong>Don’t have it—Risk Assessment;</strong> web app tool questionnaire you can (determine your risk factors and actions you should take)
<p>
<strong>Don’ have it—Symptom Tracker; </strong>Day-by-day symptoms diary tool—triggers, depending on your input, to get advice from a call center, etc.
<p>
<strong>Don’ have it—Preventive measures;</strong> Collection of fun, informative and shareable tools such as “Serious Do’s, funny Don’ts—do wash your hands, Don’t put your hand in alligator mouth” and “Bingo card to stamp what you’ve done—<a href="https://docs.google.com/document/d/1vGH_3vNtkbDcy9YPdzILc3mMx2LoSznAJwtS-dcwB_s/edit#bookmark=id.jfbimwql5pzs">filled out with top preventive  measures</a><span style="text-decoration:underline;"> </span>
   </td>
   <td>Community leaders who reach out to their communities.
<p>
POC—younger generation within multi-generational family homes 
   </td>
   <td>Increased in awareness and adoption of tracing efforts
   </td>
   <td>A 60%+ level of tool *adoption; individual use of the intake tool and a decrease of COVID-19 transmission by half.
   </td>
  </tr>
  <tr>
   <td>We will create a reward for those who share their status in our app, by giving them a “I shared my status” sticker and or editable social avatar…
   </td>
   <td>Community leaders who reach out to their communities.
<p>
POC—younger generation within multi-generational family homes 
   </td>
   <td>Increased in adoption of tracing efforts
   </td>
   <td>A 60%+ level of tool *adoption; individual use of the intake tool and a decrease of COVID-19 transmission by half.
   </td>
  </tr>
  <tr>
   <td>We will create a referral program for those who “I’ve shared my status” in the app, by giving them an incentive to refer the app to others with a prompt to have their friends share their status…
   </td>
   <td>Community leaders who reach out to their communities.
<p>
POC—younger generation within multi-generational family
   </td>
   <td>Increased in adoption of tracing efforts
   </td>
   <td>A 60%+ level of tool *adoption; individual use of the intake tool and a decrease of COVID-19 transmission by half.
   </td>
  </tr>
</table>

**Comment by jeremybrady** [comment]
**Congrats on finishing round 2 of our Design Studio!**

**A. Next steps for the team**
@CheetoMao @bmccrossen @HauwaAguillard @vegasgeek @BenRongey @alinelea
Team—On individual sticky notes, list the highlights your [sketched idea posted in Mural.](https://app.mural.co/t/nightingale1567/m/nightingale1567/1590690171105/d56b4e0fc5ecb3d6bfc523ba5e8007282b30e731) 
See column 3 under Round 2 and highlights to the row that your sketch sits on.
*This can be done by others in the team while the person who made the sketch explains his or her idea.  

B. Next Steps for @Mcscott142 —[Groom the validation board](https://docs.google.com/spreadsheets/d/1ih9t6cCNgddM-vwEediRvrWp7QaZ1DbxvVQ1FqEWzkM/edit#gid=0)
- Prioritize hypothesis
- Shore up testing methods 
- Shore up measurement numbers and tooling
- Define minimal success criteria

**C. Plan the sprint/agile cycle**
- Write epic and stories for hypothesis
- Team kickoff

**Comment by Mcscott142** [comment]
Closing this one. Work continues with #55
